testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/security/xacml/EjbXACMLAuthorizationModuleTestCase.java

```diff
@@ -160,7 +163,13 @@ private static JavaArchive createJar(final String archiveName, final String secu
                 .addAsResource(EjbXACMLAuthorizationModuleTestCase.class.getPackage(),
                         XACMLTestUtils.TESTOBJECTS_POLICIES + "/ejb-xacml-policy.xml", "xacml-policy.xml")
                 .addAsManifestResource(EjbXACMLAuthorizationModuleTestCase.class.getPackage(),
-                        XACMLTestUtils.TESTOBJECTS_CONFIG + "/jboss-ejb3.xml", "jboss-ejb3.xml");
+                        XACMLTestUtils.TESTOBJECTS_CONFIG + "/jboss-ejb3.xml", "jboss-ejb3.xml")
+                .addAsResource(createPermissionsXmlAsset(
+                        new RuntimePermission("accessDeclaredMembers"),
+                        new RuntimePermission("getClassLoader"),
+                        new ElytronPermission("getSecurityDomain"),
+                        new ReflectPermission("suppressAccessChecks")
```

Comment:
Please remove the `RuntimePermission` from here - there will be added privileged block instead:
https://github.com/picketbox/security-xacml/pull/10
(discussion in https://issues.jboss.org/browse/WFLY-10190 )

Reply:
@hkalina done, I've checked it with the latest XACML version including your picketbox/security-xacml#10 PR and those permissions are unnecessary. Thanks for pointing it out.